tests/typecheck/test_import_all.yml

```diff
@@ -231,6 +231,7 @@
         import django.db.backends.dummy.base
         import django.db.backends.mysql
         import django.db.backends.mysql.client
+        import django.db.backends.oracle.client
```

Comment:
I noticed these two files weren't being checked as part of `pytest tests/typecheck/test_import_all.yml::import_all_modules` so I added them as I was typing those files. However there's quite a few missing modules compared to Django 3.2/4.0, I didn't want to muddy up this PR, but is there an automated way to update this test for better coverage?

Reply:
> but is there an automated way to update this test for better coverage?

No 🙂